test/higher_order_ops/test_with_effects.py

```diff
@@ -86,11 +85,13 @@ def forward(self, x):
         self.assertExpectedInline(
             str(gm.code).strip(),
             """\
-def forward(self, arg0_1):
+def forward(self, arg0_1, arg1_1):
     _tensor_constant0 = self._tensor_constant0
-    takes_foo = torch.ops._TorchScriptTesting.takes_foo.default(_tensor_constant0, arg0_1);  _tensor_constant0 = None
-    add = torch.ops.aten.add.Tensor(arg0_1, takes_foo);  arg0_1 = takes_foo = None
-    return (add,)""",  # noqa: B950
+    with_effects = torch._higher_order_ops.effects.with_effects(arg0_1, torch.ops._TorchScriptTesting.takes_foo.default, _tensor_constant0, arg1_1);  arg0_1 = _tensor_constant0 = None
```

Comment:
I'm confused --  how did we get this working without Yidi's PR over at https://github.com/pytorch/pytorch/pull/120045? IMO if the there's no abstract impl for a class, then exporting should error out

Reply:
This is just with aot_export_module, aka suo's old hacks